wa/framework/plugin.py

```diff
@@ -384,6 +384,16 @@ class TargetedPlugin(Plugin):
     """
 
     suppoted_targets = []
+    parameters = [
+        Parameter('cleanup_assets', kind=bool,
+                  global_alias='cleanup_assets',
+                  aliases=['clean_up'],
+                  default=True,
+                  description="""
+                  If ``True``, assets that are deployed or created as part of the
+                  workload will be removed again from the device.
```

Comment:
Should "workload" now be changed to something more generic?

Reply:
Good point. Have updated.